platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/TelemetryDefinition.java

```diff
@@ -37,4 +39,11 @@
    * @return true if rotation session id was updated
    */
   boolean setSessionIdRotationInterval(int interval);
+
+  /**
+   * Register an end-user offline download event.
+   *
+   * @param offlineDefinition the offline region definition
+   */
+  void onCreateMapOfflineRegion(OfflineRegionDefinition offlineDefinition);
```

Comment:
nit: rename to onCreateOfflineRegion, Map is not a relevant keyword in this context. A user can also create an offline region without a map.